gapic-generator-cloud/Rakefile

```diff
@@ -43,6 +43,7 @@ namespace :gen do
   task :templates do
     generate_cloud_templates "speech"
     # generate_cloud_templates "vision"
+    # generate_default_templates "garbage"
```

Comment:
The cloud generator currently breaks when trying to generate anything other than speech, because so much of it is faked out and specific to speech. The gapic generator is able to build all the inputs, so it isn't commented out.